recipes/process-compose/recipe.yaml

```diff
@@ -0,0 +1,69 @@
+context:
+  version: "1.46.0"
+
+package:
+  name: process-compose
+  version: ${{ version }}
+
+source:
+  url: https://github.com/F1bonacc1/process-compose/archive/refs/tags/v${{ version }}.tar.gz
+  sha256: 709399d33dbc1d9ec3bf3bbf4533bf5a2351e57ece76a2f645fa8463859bb80c
+  target_directory: src
+
+build:
+  number: 0
+  script:
+    - cd src
+    - go-licenses save ./src --save_path ../library_licenses
+    - if: win
+      then: go build -v -o %LIBRARY_BIN%\process-compose.exe ./src
+      else: go build -v -o $PREFIX/bin/process-compose ./src
+    - if: unix
```

Comment:
It’s probably better to have a separate package with completions that can be installed on osx-arm64

Reply:
As discussed, we’ll just build natively with the new osx-arm64 runner